x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/http/sender/HttpRequestSender.java

```diff
@@ -73,7 +73,7 @@ public Sender createSender(String serviceName) {
     private final HttpClientManager manager;
     private final RequestExecutorService service;
     private final AtomicBoolean started = new AtomicBoolean(false);
-    private final CountDownLatch startCompleted = new CountDownLatch(2);
+    private final CountDownLatch startCompleted = new CountDownLatch(1);
```

Comment:
Why is the countdownlatch decreasing from 2 to 1?

Reply:
Yeah it's because I removed a `countDown()` call on line 106